src/databases/databases.js

```diff
@@ -15,6 +15,13 @@ if (config.createDatabaseIfNotExist && !config.readOnly) {
   if (fs.existsSync(config.privateDBSchema)) privateDB.exec(fs.readFileSync(config.privateDBSchema).toString());
 }
 
+// Upgrade database if required
+let versionCode = db.prepare("SELECT code FROM version").get() || 0;
+let path = config.schemaFolder + "/_upgrade_" + versionCode + ".sql";
```

Comment:
This should get the current schema version from the db, then get the latest upgrade file, then iterate over them. That way, somebody updating from say 3 to 5 will get upgrade 4 and 5.